R/bambu-processReads_utilityConstructReadClasses.R

```diff
@@ -64,6 +61,10 @@ constructSplicedReadClasses <- function(uniqueJunctions, unlisted_junctions,
 
     unlisted_junctions <- correctIntronRanges(unlisted_junctions, 
         uniqueJunctions, correctedJunctionMatches)
+    toRemove = which(mcols(unlisted_junctions)$remove)
+    unlisted_junctions = unlisted_junctions[-toRemove]
+    allToUniqueJunctionMatch = allToUniqueJunctionMatch[-toRemove]
+    
```

Comment:
this code will fail for data sets without microexons (please test/ run unit tests)
```suggestion
    if(any(mcols(unlisted_junctions)$remove){
    toRemove = which(mcols(unlisted_junctions)$remove)
    unlisted_junctions = unlisted_junctions[-toRemove]
    allToUniqueJunctionMatch = allToUniqueJunctionMatch[-toRemove]
    }
```